source/target/nordic/target_reset.c

```diff
@@ -45,24 +45,55 @@ uint8_t target_set_state(TARGET_RESET_STATE state)
 
 void swd_set_target_reset(uint8_t asserted)
 {
+    extern char *board_id;  //TODO - remove
```

Comment:
I think the goal was to move this to a header file that is shared between info.c and the board.  For here I would recommend including info.h and calling info_get_board_id.